docs/.vuepress/components/TutorialList.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="tutorial-container">
+    <div v-for="(t, index) in tutorials" :key="t.title" class="tutorial-card">
+      <h3 class="tutorial-title">
+        <a :href="`${tutorialsRepo}${t.slug}/TUTORIAL.md`" target="_blank">{{ t.title }}</a>
```

Comment:
Should this be a link to `github.com` and not `githubercontent.com`?

Reply:
I just learnt this from GPT now:

The difference between the `raw.githubusercontent.com` domain and the `github.com` domain is in the purpose and the type of content they serve:

1. **raw.githubusercontent.com**:
   - **Purpose**: This domain is used to serve raw files directly from GitHub.
   - **Content Type**: When you access a file (e.g., a `.json`, `.txt`, `.js`, etc.) via this domain, it serves the file as plain text without any additional formatting or styling. This makes it suitable for programmatic access, like fetching the content of a file in scripts or applications.
   - **Headers**: The response headers for this domain are set in a way that tells the browser to treat the content as raw data. For example, a JavaScript file will be served as `text/plain` instead of `application/javascript`, preventing it from being executed as a script in the context of a website.

2. **github.com**:
   - **Purpose**: This domain is used for the main GitHub website, where users can browse repositories, view files, manage issues, and perform other GitHub-related activities.
   - **Content Type**: When you access a file via this domain, it's presented within the GitHub UI with syntax highlighting, line numbers, and other features. It's designed for human readability within a web interface, not for programmatic access.
   - **Headers**: The content served from this domain comes with headers suitable for displaying within the GitHub website's context.

For your use case, where you want to fetch the content of a `tutorials.json` file programmatically, using `raw.githubusercontent.com` is more appropriate. Fetching from the `github.com` domain would give you the entire HTML page, including all the GitHub UI elements, which isn't what you want when you're trying to access the raw content of a file for processing in a script or application.

Reply:
Thanks for pointing out! Fixed!